### DISCUSSION TITLE
"Previous" and "Next" page navigation

### DISCUSSION THREAD

**Discussion by Arlene-Xu-LL** [RedHat-UX/red-hat-design-system#936] [Ideas]
Add "Previous" and "Next" page navigation buttons at the bottom of each page to improve the user experience by providing a more seamless reading and navigation flow. 

This can be achieved by creating a new component in the design system and integrating it into the website templates. This feature will make it easier for users to navigate between pages and find related content, resulting in a better overall user experience.

**Comment by bennypowers** [comment]
do previous and next just follow the linear top-to-bottom order of the side nav (presuming everything is open)?

What if we pivoted from side nave to navigation-secondary, then I suppose it would be left-to-right, top-to-bottom order?
